cmd/blockchaincmd/add_validator.go

```diff
@@ -128,18 +129,36 @@ func addValidator(_ *cobra.Command, args []string) error {
 		return err
 	}
 
+	sc, err := app.LoadSidecar(blockchainName)
```

Comment:
this is the same code 3 times. Please move it to separate func maybe in utils or in app.
This is very useful functionality and might be even expanded later to more commads.